m01_introduction/index.rst

```diff
@@ -5,12 +5,11 @@ Introduction to developing course on A-Plus learning management service.
 
 .. toctree::
 
-  01_installation
-  0X_gallery
-  02_rst
-  03_css
-  04_git
-  05_docker
-  06_languages
-  07_questionnaires
+  01_gallery
+  02_languages
+  03_questionnaires
   enrollment_en
+
+.. aplusmeta::
+  :introduction: <h3>Introduction</h3>
```

Comment:
What is the idea behind these module introductions that repeat the module title and the phrase "cover the following topics"?